resources/sass/app.scss

```diff
@@ -64,4 +64,9 @@ ul {
 .modal-body {
     max-height: calc(100vh - 200px);
     overflow-y: scroll;
+}
+
+.datepicker {
```

Comment:
I don't think this solves the issue with the datepicker. Its still the same on my end. Maybe increase the modal width ?